node-template/src/service.rs

```diff
@@ -92,6 +95,9 @@ construct_service_factory! {
 				Ok(service)
 			}
 		},
+		AuthorityKeyProvider = () {
+			|_, _| Ok(())
```

Comment:
Not convinced about this. The `node-template` should really have proper configuration for offchain workers.